app/models/miq_shortcut.rb

```diff
@@ -20,8 +26,7 @@ def self.seed
       end
     end
 
-    all.each do |rec|
-      next if names.include?(rec.name)
+    db_data.each do |name, rec|
```

Comment:
Yeah, I see what you are doing now, thanks for the clarification.  Looks good.  👍